lib/promotion.rb

```diff
@@ -74,7 +78,6 @@ def change_trust_level!(level, opts = {})
       @user.save!
       @user.user_profile.recook_bio
       @user.user_profile.save!
-      DiscourseEvent.trigger(:user_promoted, user_id: @user.id, new_trust_level: new_level, old_trust_level: old_level)
```

Comment:
I think we should just leave the trigger as it is as other plugins might be depending on this event. If we want to remove it, we'll have to depreciate it first.